### DISCUSSION TITLE
making do with git?

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#292] [Q&A] [answered]
@orasanen asks:
> Do you know how to replace "datalad get . # get everything" with a corresponding git command when trying to get all the data for https://gin.g-node.org/ACLEW/ACLEW ? Git pull in correct sub-folders does nothing.

The context is that he and his postdoc haven't been able to get datalad installed because their OS is too old or the machine is too small.

**Comment by orasanen** [comment, marked as answer]
The problem in this case is just that git-annex does not compile for MacOS 10.13, so I'd need to do direct git pull for the datasets. Machine size is not relevant to this query.

**Reply by lucasgautheron** [reply]
Hello, can you confirm that you have tried with `brew install git-annex` ? It usually works, except on very old mac versions, for which no binaries are provided and compilation fails too.

If it is indeed `brew install git-annex` that fails because the OS is too old, I suggest upgrading MacOS if possible (we had to do that once for an old laptop at the lab)

**Reply by orasanen** [reply]
Yeah I tried with brew, but my OS is too old (10.13.6). I will need to get the OS updated, but is unfortunately not doable on this machine any time soon. In any case, I have newer OS on my other machine I can hopefully get back to next week, so this problem goes away with that, and lets not waste more (of your) time on this.

**Comment by alecristia** [comment]
ah got it! in that case, perhaps the only option is to direct download from gin (via the browser -- ie, a pain)?

and for Maria Cruz, what is the installation problem?

**Reply by orasanen** [reply]
I dont think she has any problems with installation.

**Reply by orasanen** [reply]
I'll continue on this next week, when I can hopefully get back to lab & access my main computer.
